edx_django_utils/plugins/docs/decisions/0003-plugin-contexts.rst

```diff
@@ -0,0 +1,88 @@
+Plugin Contexts
+---------------
+
+Status
+======
+Draft
+
+Context
+=======
```

Comment:
Add spaces after headers.

Reply:
I assume you meant a new line after header? correct me if I am making the wrong assumption.